test/sparse/sparse.jl

```diff
@@ -1674,168 +1596,9 @@ end
 # 19304
 @inferred hcat(sparse(rand(2,1)), eye(2,2))
 
-# Test that broadcast[!](f, [C::SparseMatrixCSC], A::SparseMatrixCSC, B::SparseMatrixCSC)
-# returns the correct (densely populated) result when f(zero(eltype(A)), zero(eltype(B))) != 0
```

Comment:
The `broadcast[!] implementation specialized for pairs of (input) sparse vectors/matrices` testset replaces those tests.